.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": [
+    "latest"
+  ],
+  "plugins": [
+    "add-module-exports",
+    "transform-flow-strip-types"
```

Comment:
K. I also removed all the dependencies and rollup config code related to flow as well.